script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -361,26 +366,26 @@ Then finally, began the autumn of my second year of high school.
 [sha:46a8dc4d84ed6af12630d7a56f8e75e9bfc49374]{
 -- Page 26, Offset 1216.
 -- あの時―――別れ際に先生が言っていたような特別な出来事はまったく起こらなかったし、自分も先生のくれた眼鏡をつけているかぎり『線』を見る事はない。
-The special event that Sensei told me of when we parted never occurred,
+The kind of special incident that Sensei told me about hasn't occurred in the slightest,
  and as long as I wear the glasses she gave me,
  I don't see those lines at all.
 }
 [sha:d314fe5bb6919f436e2c107165f40ad3c1d0ba1f]{
 -- Page 26, Offset 1217.
 -- 遠野志貴は平凡ではあるものの、得難い日々を過ごしてきたと思う。
-I believe that I've been spending my days living a precious, yet ordinary life.
+I believe that I've been spending my days living a precious, yet ordinary way.
```

Comment:
I think what you meant was 'spending my days living in a precious, yet ordinary way'.

With that said, I also prefer original here. I think the emphasis on him having lived a 'normal life' is what's important here.

Reply:
Also prefer orig